docs/local-dev/debugging.md

```diff
@@ -71,3 +71,17 @@ In your `launch.json`, add a new entry with the following,
     "console": "integratedTerminal"
 },
 ```
+
+### WebStorm
```

Comment:
Could we add a short sentence here: "Here's how to setup WebStorm to debug your Backstage instance:"

And then change this to a numbered list? Feels like it would read better.